.github/workflows/tests.yaml

```diff
@@ -9,16 +9,16 @@ jobs:
         include:
           - _defaults: # see Dockerfile for variable defaults
           - TASK_VERSION: v2.4.4
-          - TASK_VERSION: 2.5.2
-            continue-on-error: true
+          - TASK_VERSION: v2.5.1
```

Comment:
I guess we want this to be `v2.5.2.`?

Reply:
Actually, scrolled down, realized that's the default now with this PR :)

Reply:
Yeah, that's the strategy here: keep reasonable defaults in the Dockerfile for local testing, everything else in the matrix.